init.lua

```diff
@@ -608,7 +642,7 @@ local function tool_active(player, item)
 	local inv = player:get_inventory()
 	local hotbar = player:hud_get_hotbar_itemcount()
 	for i=1, hotbar do
-		if inv:get_stack("main", i):get_name() == item then
+		if inv:get_stack("armor", i):get_name() == item then
```

Comment:
This slot is not guaranteed to exist, hence breaks games without armor slots.

Reply:
fixed